src/lightning/data/streaming/fields.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+
+from torch import Tensor
+
+
+@dataclass
+class Data:
+    pass
```

Comment:
why do you need this base-class?

Reply:
I was exploring something but not needed right now. Removing it.